monai/deploy/operators/dicom_data_loader_operator.py

```diff
@@ -90,7 +91,10 @@ def _load_data(self, files: List[str]):
         sop_instances = []
 
         for file in files:
-            sop_instances.append(dcmread(file))
+            try:
+                sop_instances.append(dcmread(file))
+            except InvalidDicomError:
+                print(f"Failed to read {file}; file may not be a valid DICOM file")
```

Comment:
This can use logging.warning instead of print. Also, since the specific InvalidDICOMError is handled and not propagating up, it would be better to state the precise meaning of the error, "Failed to read a DICOM dataset from {file} because the dataset is not stored in accordance with the DICOM File Format."